classes/shortcodes.php

```diff
@@ -216,20 +252,28 @@ function do_shortcode( $args ) {
 					switch ( $tax_format ) {
 						case 'csv':
 						default:
-							return ! empty( $term_names ) ? $args['before'] . implode( ', ', $term_names ) . $args['after'] : '';
+							return ! empty( $term_names ) ? $args['before']
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->